lib/ansible/modules/cloud/digital_ocean/_digital_ocean.py

```diff
@@ -185,7 +189,7 @@
 
 try:
     # Imported as a dependency for dopy
-    import six
+    import ansible.module_utils.six as six
```

Comment:
You can fix the last Shippable failure by changing this to `from ansible.module_utils import six`.

Reply:
Yea I thought I made this change but it appears I didn't push my code from a different machine. Taken care of in the latest commit and I removed the [WIP] title. Just waiting on the other module now.